### DISCUSSION TITLE
AKHQ with JDK 17

### DISCUSSION THREAD

**Discussion by AdiWehrli** [tchiotludo/akhq#1462] [General]
Hello

Is it planned to upgrade AKHQ to JDK 17 (resp. Java 17)?

Thanks a lot in advance.

Kind regards

**Comment by tchiotludo** [comment]
THere is this [PR](https://github.com/tchiotludo/akhq/pull/1279) about java 17. 
I think next release will be java 17 only, since micronaut 4 will be out with java 17 as default version

**Comment by AdiWehrli** [comment]
Hello @tchiotludo 

Thanks for your and the fellow contributor's good work.

What's the state about Java 17 currently?

Thanks very much for your repsonse.

Kind regards,
Adi

**Reply by tchiotludo** [reply]
done on the `dev` and release soon ;)
